### PR TITLE
LVM-261  Tests for Coordinator Controller

### PR DESCRIPTION
**Pull request by jlally21**
https://cs5500-jira.ccs.neu.edu/browse/LVM-261

### CODE REVIEW

**Review by vcantu — APPROVED** [review]

File: spec/controllers/coordinator_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+      before do
+        @coordinator = create(:coordinator)
+      end
+
+      it 'populates the specified coordinator' do
+        get :show, id: @coordinator
+        expect(assigns(:coordinator)).to eq(@coordinator)
+      end
+
+      it 'populates an enrollment' do
```
Comment (zack) [inline spec/controllers/coordinator_controller_spec.rb]: It doesn't populate an enrollment! But it *does* populate students and tutors!
I suggest moving the `students` and `tutors` methods into the `private` part of this controller and then testing that the `show` view populates students and tutors. The easiest way to do this is probably to create a factory for it, but you can also definitely set it up manually! Let me know if I can help.

To make private methods, you just move the method definitions underneath the `private` keyword near the bottom of the controller, so the methods would move either directly under or directly over the `coordinator_params` method.

**Comment by zack** [comment]
Very nice! one comment but looks good.

**Comment by zack** [comment]
@jlally21 just one comment up above there.

**Comment by jlally21** [comment]
@zack see new commit.  Could you help me with the factory creation part?  Not really sure what to do...

**Pull request merged**